Week2/todo_app/program.js

```diff
@@ -2,6 +2,7 @@
 // that we can use to connect to a MySQL process
 // and access its databases.
 const mysql = require('mysql');
+const config = require('./config.js');
```

Comment:
Try and only include the relevant homework in a PR - you don't need to include week2 code for week3 homework.